src/qibolab/_core/dummy/platform.py

```diff
@@ -1,9 +1,9 @@
 import pathlib
 
-from qibolab.components import AcquisitionChannel, DcChannel, IqChannel
-from qibolab.instruments.dummy import DummyInstrument, DummyLocalOscillator
-from qibolab.platform import Platform
-from qibolab.qubits import Qubit
+from qibolab._core.components import AcquisitionChannel, DcChannel, IqChannel
+from qibolab._core.instruments.dummy import DummyInstrument, DummyLocalOscillator
+from qibolab._core.platform import Platform
+from qibolab._core.qubits import Qubit
```

Comment:
Ah, yes, that's a pain point.

This is something I also discussed before. Before, I was always using relative imports, with the rationale to avoid naming explicitly the package.

By now I tried to balance it a bit more: `qibolab`'s name is stable as its content (actually, much more), and the idea was to avoid navigating too much in both directions.

If it is something in the exact same subpackage, it has to be a relative import, also because it might be move altogether, and it will maintain consistency.
If it's a module deeply nested, importing something that diverge at root level, it makes more sense to use the absolute routing.

The problem is: I don't know any longer where to put the threshold...

Reply:
I see, but in terms of this specific file (`_core/dumy/platform.py`), what is the difference from let's say (`_core/compilers/default.py`)? They are both depth-2 (counting from `_core`) and in compiler you use relative import
```py
from ..qubits import Qubit
```
while here we are using absolute import.

Reply:
> I see, but in terms of this specific file (`_core/dumy/platform.py`), what is the difference from let's say (`_core/compilers/default.py`)? They are both depth-2 (counting from `_core`) and in compiler you use relative import

The line of reasoning was about that mutual relationships between those modules *may* be more prone to changes.

> while here we are using absolute import for `Qubit` (and the rest of objects).

Well, that's the point, I don't have a rule. That's why is confused... (essentially because I am...)

> In any case, this is pretty minor and fully internal.

This is more or less the solution and the problem at the same time. Since it's quite irrelevant, it's actually fine both ways. But since it's not that relevant, I don't have a proper way to decide what to do...

In any case, feel free to change it, and even to propose a rule yourself, if you find a meaningful one.
This is like `isort` for the imports, or Black for the format: I don't care much, I'd just like to have one sensible behavior and stick to that. But I'd care not to change it over and over...